packages/embark/src/lib/modules/contracts_manager/index.js

```diff
@@ -299,6 +300,25 @@ class ContractsManager {
           );
         callback(null, allContractsCompiled);
       },
+      function filterUnchangedContracts(allContractsCompiled, callback) {
+        let contractHashMap = fs.existsSync(CONTRACTS_HASHES_FILE) ? require(CONTRACTS_HASHES_FILE) : {};
+
+        async.each(self.contractsFiles, function (contractFile, cb) {
```

Comment:
Wanted to use `async/await` here but it doesn't work nicely with `async` library